x-pack/plugins/observability_solution/observability/public/pages/slo_edit/components/common/query_documents_flyout.tsx

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I would appreciate if you could show the index pattern for the documents you're looking at somewhere in the flyout.

![image](https://github.com/elastic/kibana/assets/11356435/aace050e-eaba-4c61-8391-469cb6639336)

Reply:
Done 
<img width="1728" alt="image" src="https://github.com/elastic/kibana/assets/3505601/0d1d07e6-18ba-40d3-a206-01814c90fafe">